api-reference/10 UI Widgets/dxScheduler/1 Configuration/cellDuration.md

```diff
@@ -5,6 +5,6 @@ default: 30
 ---
 ---
 ##### shortDescription
-Specifies cell duration in minutes.
+Specifies cell duration in minutes. This property's value should divide the interval between [startDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#endDayHour) into even parts.
```

Comment:
```suggestion
Specifies cell duration in minutes. This property's value should evenly divide the interval between [startDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#endDayHour).
```

Reply:
What do you mean by "cell duration"? Can you explain?

Reply:
Cell duration denotes the amount in minutes allocated per cell. This value affects the visual representation of cells within the Scheduler.

Reply:
Ok